site/content/en/docs/Examples/simple-genai-gameserver.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Example Build and Run a Simple Gameserver that Connects to an Inference Server"
+linkTitle: "Build and Run a Simple GenAI Gameserver"
+date:
+publishDate:
+description: >
+  This example shows how to set up a simple game server that integrates with your inference server's GenAI endpoints. You can either interact with one GenAI endpoint via TCP, or set up two endpoints to "chat" to each other.
+---
+
+## Objectives
```

Comment:
My personal opinion, but I don't think we need objectives in the examples documentation (unlike in the getting started tutorials).